Example/Emission/AppDelegate.m

```diff
@@ -254,6 +255,10 @@ - (UIViewController *)viewControllerForRoute:(NSString *)route;
     NSString *artistID = [[route componentsSeparatedByString:@"/"] lastObject];
     viewController = [[ARArtistComponentViewController alloc] initWithArtistID:artistID];
 
+  } else if ([route hasPrefix:@"/show/"] || [route hasPrefix:@"show/"]) {
```

Comment:
This displays our new show screen instead of the web view, right?

Reply:
In the Emission app -  it's a blank screen that says the path, so "/show/whatever" (in Eigen it would show the internal web view)

Reply:
Well this hooks up our ShowVC view in emission so if you tap on a show it'll render it instead of the blank screen. Will have to register ShowVC to show urls in Eigen next